site/docs/components/text/examples.mdx

```diff
@@ -14,55 +14,55 @@ data:
 
 ### Styles
 
-The Text component supports these sets of styles:
+`Text` supports these sets of styles:
 
 - Display
 - Headings
 - Body
 - Label
 
-These alternative components display text in commonly used styles. Where relevant, use them instead of Text to simplify syntax:
+These alternative components display text in commonly used styles. Where relevant, use them instead of `Text` to simplify syntax:
 
-- Display1
-- Display2
-- Display3
-- H1
-- H2
-- H3
-- H4
-- Label
-- TextNotation
-- TextAction
+- `Display1`
+- `Display2`
+- `Display3`
+- `H1`
+- `H2`
+- `H3`
+- `H4`
+- `Label`
+- `TextNotation`
+- `TextAction`
 
 </LivePreview>
 
 <LivePreview componentName="text" exampleName="Truncation">
 
 ### Truncation
 
-The component wraps by default, but you can use the `maxRows` prop to force truncation. Truncation will occur when `maxRows` is greater than 0. If a value is not provided, the text content will wrap.
+The component wraps by default, but you can use the `maxRows` prop to force truncation. Truncation will occur when `maxRows` is greater than 0. If you don't provide a value, the text content will wrap.
 
-#### Guidance
+#### Best practices
 
-Truncated text must comply to WCAG SC 1.4.4. If you use truncated text, you must ensure that a screen reader can read the full value. Read the Accessibility section for further guidance.
+Truncated text must comply with WCAG SC 1.4.4. If you use truncated text, you must ensure that a screen reader can read the full value. Read the Accessibility section for further guidance.
```

Comment:
This is also in the accessibility section but with a link. Should we add the same link here?